modules/consul-iam-policies/variables.tf

```diff
@@ -6,3 +6,8 @@
 variable "iam_role_id" {
   description = "The ID of the IAM Role to which these IAM policies should be attached"
 }
+
+variable "enabled" { 
+  description = "Give the option to disable this module if required"
+  default = true
```

Comment:
Please run `terraform fmt` on all of the code.

Reply:
Usually i have pre-commit to do this.  I will add the pre-commit.yml to do this on commit if you have no issue with it?  In the meantime i will run fmt on it.